test/test_worker.py

```diff
@@ -24,6 +24,24 @@ def test_basic(self):
             'stalled': 0
         }])
 
+    def test_worker_registration_lapse(self):
+        '''Workers that fail to check in expire from the list of active workers'''
+        self.lua('config.set', 0, 'max-worker-age', 10)
+        self.lua('pop', 1, 'queue', 'worker', 1)
```

Comment:
It looks like you're popping a job without doing a put first. Will this exercise the code path we're trying to test?

Reply:
This test is just to prove that `pop` does work to mark a worker as active, and does not exercise the `heartbeat` codepath. Getting a job back from the `pop` is unimportant for this case, but in the other test it's necessary so that it has something to call `heartbeat` on.